content/en/opentelemetry/interoperability/instrumentation_libraries.md

```diff
@@ -22,19 +22,24 @@ This provides observability for libraries not covered by Datadog SDKs without ch
 
 ## Prerequisites
 
-Before adding OpenTelemetry instrumentation libraries, set the `DD_TRACE_OTEL_ENABLED` environment variable to `true`.
+1. **Enable OpenTelemetry support**: Set the `DD_TRACE_OTEL_ENABLED` environment variable to `true`. This step isn't required for the Datadog Go and Ruby APM SDKs.
```

Comment:
All of these links are broken. I think it might be fixed by merging the latest in from master, since we made some updates for the Python and Ruby drop-in support PR.

This should be at the bottom of the page:
```
[1]: /tracing/trace_collection/
[2]: /tracing/trace_collection/automatic_instrumentation/
[3]: https://opentelemetry.io/docs/concepts/instrumentation/libraries/
[12]: /opentelemetry/interoperability/otlp_ingest_in_the_agent/?tab=host
[16]: /opentelemetry/interoperability/environment_variable_support
[24]: /getting_started/tracing/#set-up-datadog-apm
```

Reply:
After rebasing, these changes should no longer be present. Going to close for now